_posts/2017-09-25-bagwell-award-2017.md

```diff
@@ -0,0 +1,39 @@
+---
+category: news
+title: "Josh Suereth receives community award"
+---
+
+We are pleased to announce that the Phil Bagwell Memorial Scala Community Award for 2017 has been awarded to [Josh Suereth](https://twitter.com/jsuereth).  The award was presented in England's Lake District at [Scala World 2017](https://scala.world).
```

Comment:
I'm totally guilty of doing this, but let's say "the UK's" rather than "England's". England isn't really a country... ;)

Reply:
you patriot! 🇬🇧🇬🇧🇬🇧

changed.